licenses/license-engine.sh

```diff
@@ -25,9 +27,12 @@ ALLOWED_LICENSES=(
 	'CC-BY-4.0'
 	'(CC-BY-4.0 AND MIT)'
 	'ISC'
+	'ISC*'
 	'LGPL-2.1' # LGPL allows commercial use, requires only that modifications to LGPL-protected libraries are published under a GPL-compatible license
 	'MIT'
 	'MIT*'
+	'MIT-0'
+	'MIT AND ISC'
```

Comment:
Since both MIT and ISC are fine, these should be fine, too.

Reply:
Jonas did the work. I just quickly reviewed the licenses, and it looks good.